salt/cloud/deploy/bootstrap-salt.sh

```diff
@@ -227,7 +227,7 @@ _SALT_MASTER_ADDRESS=${BS_SALT_MASTER_ADDRESS:-null}
 _SALT_MINION_ID="null"
 # _SIMPLIFY_VERSION is mostly used in Solaris based distributions
 _SIMPLIFY_VERSION=$BS_TRUE
-_LIBCLOUD_MIN_VERSION="0.14.0"
+_LIBCLOUD_MIN_VERSION="2.0.0"
```

Comment:
I think that's the question for @dmurphy18.

That line in `bootstrap-salt.sh` actually only makes sense when you install `salt-cloud` itself with `pip` using the script, not `salt-minion`. So, that's not a big deal though.
I have brought attention to it, because I know this particular script is just copied from [salt-bootstrap](https://github.com/saltstack/salt-bootstrap) repo during stable releases.
There is a high chance that your change will be just overwritten :smile: